pkgs/by-name/ja/ja2-stracciatella/package.nix

```diff
@@ -46,17 +44,12 @@ stdenv.mkDerivation {
     cmake
     python3
   ];
-  buildInputs =
-    [
-      SDL2
-      fltk
-      rapidjson
-      gtest
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      Carbon
-      Cocoa
-    ];
+  buildInputs = [
+    SDL2
+    fltk
+    rapidjson
+    gtest
+  ];
```

Comment:
These formatting changes look like they shouldn't be part of the commit that moves the package to by-name?

Reply:
Removing the darwin cruft, should go together with the move, IMO, since it should be a NOOP.
And I wanted every commit to be correctly formatted.

I can change this if you want, but I'd rather not waste time on it.

Reply:
Maybe start with a formatting commit then.  It's very surprising to see any diff in a move-to-by-name commit.

Reply:
I'll try